reactivexcomponent/reactivexcomponent/communication/publisher.py

```diff
@@ -0,0 +1,96 @@
+""" Publisher """
+
+import json
+import websocket
+from reactivexcomponent.configuration.serializer import *
+from lxml import etree
+
+class Publisher:
+	
+	def get_xml_content(self):
+		tree=etree.parse(self.file)
```

Comment:
lint? 

i added a comment on missing spaces before... and they are still missing.